src/validation/trust.information.validation.ts

```diff
@@ -1,7 +1,12 @@
 import { body } from "express-validator";
+import { checkAtLeastOneFieldHasValue, checkMandatoryTrustFields } from "./custom.validation";
 
 import { ErrorMessages } from "./error.messages";
 
 export const trustInformation = [
-  body("trusts").not().isEmpty({ ignore_whitespace: true }).withMessage(ErrorMessages.TRUST_DATA_EMPTY)
+  body("trusts").not().isEmpty({ ignore_whitespace: true }).withMessage(ErrorMessages.TRUST_DATA_EMPTY),
+  body("beneficialOwners").custom((value, { req }) =>
+    checkAtLeastOneFieldHasValue(ErrorMessages.TRUST_BO_CHECKBOX, req.body.beneficialOwners)),
+  body("trusts").custom((value, { req }) =>
```

Comment:
you should be able to just chain the 'trust' checks,
body("trusts")
  .not().isEmpty({ ignore_whitespace: true }).withMessage(ErrorMessages.TRUST_DATA_EMPTY)
  .custom((value, { req }) => .....